eng/CHECKENFORCER

```diff
@@ -1,2 +1,50 @@
 format: v0.1-alpha
 minimumCheckRuns: 1
+timeout: 5
+message: >
```

Comment:
Any particular reason you decided to inline the message instead of have separate MD file like you were originally thinking? I'm fine with this approach just curious if there was a particular reason.

Reply:
I discovered the joys of multi-line YAML parsing (only half joking). But when I thought about it I decided that I already had the confg loaded in memory so there would be no need for an additional API hit to GitHub. There are some glitches in formatting which I still haven't managed to sort out with this approach, but I'm pretty sure that is my YAML-foo.